platform/web/js/libs/library_godot_audio.js

```diff
@@ -1625,6 +1663,22 @@ const _GodotAudio = {
 		return Number(GodotAudio.sampleNodes.has(playbackObjectId));
 	},
 
+	godot_audio_get_sample_playback_position__proxy: 'sync',
+	godot_audio_get_sample_playback_position__sig: 'vii',
```

Comment:
```suggestion
	godot_audio_get_sample_playback_position__sig: 'vi',
```
There is only one parameter to your function here.